src/devices/DCMotor/DCMotor.cs

```diff
@@ -36,6 +31,15 @@ protected DCMotor(GpioController controller)
         /// </summary>
         public abstract double Speed { get; set; }
 
+        /// <summary>
+        /// <see cref="GpioController"/> related with operations on pins
+        /// </summary>
+        protected GpioController Controller
+        {
+            get;
```

Comment:
Why did we turn this into a property instead of a field as it used to be? This might be an unnecessary breaking change.

Reply:
I thought that normally members should not be exposed at all, and since the name was like it should be a property, I changed it to one. If you think it should rather be a protected member, I can change that, but then it will change the name.

Reply:
Looking at the code, it should probably even be read-only.

Reply:
Yeah I agree that the original design is probably not the ideal one, I was only wondering if it would be worth it to take the breaking change here or if it was better to just leave as is in this case and make an exception to the rule. I will defer to @krwq as he is the one that originally added the binding.

Reply:
I think in general I'd say we should take all of the breaking changes in Iot.Device.Bindings
 which make the style more consistent